package/etc/conf.d/conflib/syslog/app-syslog-barracuda_syslog.conf

```diff
@@ -0,0 +1,36 @@
+block parser app-syslog-barracuda_syslog() {
+ channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index("netfw")
+                sourcetype('barracuda')
+                vendor("barracuda")
+                product("waf")
+                template("t_hdr_msg")
```

Comment:
for this one we can use t_msg_only as template there is no data in the header needed